CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 **********
 
+Changed
```

Comment:
Is there a reason you aren't bumping the version yet?

Reply:
I was also going to put in a change to record Kafka error codes and didn't feel like making two releases, but looking into it some more I'm not sure if that's something we should actually ask NR to include in their agent. So I'll go and make this a version bump.